pkg/render/render.go

```diff
@@ -59,9 +61,9 @@ func RenderDir(manifestDir string, d *RenderData) ([]*unstructured.Unstructured,
 	return out, nil
 }
 
-// RenderTemplate reads, renders, and attempts to parse a yaml or
+// Template reads, renders, and attempts to parse a yaml or
 // json file representing one or more k8s api objects
-func RenderTemplate(path string, d *RenderData) ([]*unstructured.Unstructured, error) {
+func Template(path string, d *Data) ([]*unstructured.Unstructured, error) {
```

Comment:
same here! this is rendering the templates

Reply:
Sure but it's within same render package so it's render.Render* inside render folder which makes golint unhappy

Reply:
I actually took this from the Cluster Network Operator: https://github.com/openshift/cluster-network-operator/blob/master/pkg/render/render.go

Reply:
If we want ignore some rules/files etc we will need to use something different that golint (there is no ignore option there) or ignore some fails.
otherwise we need to obey the rules.

Reply:
@mrKisaoLamb this is not true as we can have some logic in shell that parses the errors, but it's hacky and I would prefer not to.